learningequality/kolibri-design-system
# Adds props/computed props to mixin for KContentRenderer

## Description


Adds `timeSpent`, `duration` props and `forceTimeBasedProgress`, `durationBased` computed props so that we can refactor Kolibri content renderers to use unified time-tracking.

#### Issue addressed


Addresses #214 
## Steps to test

Currently, to test this PR, you'll have to use learningequality/kolibri#8037 in order to make sure the new props and computed props are working (see tech-debt items below for details).

## (optional) Implementation notes

### At a high level, how did you implement this?


Added props and computed props to the mixin used by `KContentRenderer`.

### Does this introduce any tech-debt items?


There are currently no tests in KDS to address the use of these props and computed props; however, tests exist on the individual renderers in Kolibri.

## Reviewer guidance




## Comments
